### ISSUE TITLE
Move plugin to authorization extension API

### ISSUE THREAD

**Issue by varshavaradarajan** [cnorthwood/gocd-tls-auth#1]
@cnorthwood , I see you have moved the plugin to use the authorization extension. Can you please make a new release for the plugin if it is in a working state?

**Comment by cnorthwood** [comment]
Hi Varsha,

The plugin isn't quite in a working state. I've made the changes I think I need to make according to the documentation, but I get hard to follow stack traces from GoCD when I actually try and follow the process. I've asked for some support from the GoCD community but sadly got nothing back: https://groups.google.com/forum/#!topic/go-cd/SqJKZ88-yWo. When I've got some more time I'll have another attempt at trying to track down my mistake, but right now v2.0 isn't ready.

Chris
